### ISSUE TITLE
How to get frames distances and orientation of a robot?

### ISSUE THREAD

**Issue by wbadry** [cyberbotics/community-projects#15]
Hello,
I am trying to create a tutorial of webots since this platform is amazing but resources is a bit rare and scattrered sometime.

Is there any mean to apply forward kinematics without any libraries to get the robot frame orientation and distances between each frame ? This would be beautiful that any beginner could reach a workpiece by computing the forward kinematics from frame to frame until reaching the workpiece.

All I seek is a way to get for instance Kuka or any other open chan robot frames and a way to measure the distance between each frame to caculate the transformation matrix. Thanks

**Comment by omichel** [comment]
Yes, it is very simple to implement a controller with forward kinematic in Webots as you describe. You can simply control each motor in position and use a position sensor to get position feedback. Do you need anything else?
